src/main/java/com/doyak/reflector/converter/PostConverter.java

```diff
@@ -0,0 +1,48 @@
+package com.doyak.reflector.converter;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Component;
+
+import com.doyak.reflector.domain.Post;
+import com.doyak.reflector.domain.User;
+import com.doyak.reflector.dto.request.PostRequest;
+import com.doyak.reflector.dto.response.PostResponse;
+
+@Component
+public class PostConverter {
+
+    // 생성
+    public Post toEntity(PostRequest request, User user) {
+        return new Post(user,
+                        request.getSite(),
+                        request.getLevel(),
+                        request.getTitle(),
+                        request.getContent());
```

Comment:
```suggestion
        return Post.builder()
                    .site(request.getSite)
                    .level(request.getLevel)
                    .title(request.getTitle)
                    .content(request.getContent);
```

빌더 패턴을 사용한다면, 위와 같이 converter를 변경할 수 있습니다! 해당 패턴을 사용한다면 파라미터 순서를 신경 쓸 필요가 없고, 어떤 필드에 어떤 값이 들어가는 지 명시적으로 확인할 수 있으며, 필드 추가 및 변경에도 용이합니다.

Reply:
suggestion으로 올린 부분은 builder 패턴을 보여드리기 위해 간단하게 작성한 것으로, 다른 코드 수정 및 어노테이션 / import 오류를 방지하기 위해 suggestion을 바로 업데이트 하는 것보단 파일을 직접 수정하시는 걸 권장드립니다 ...

Reply:
> 빌더 패턴을 사용한다면, 위와 같이 converter를 변경할 수 있습니다! 해당 패턴을 사용한다면 파라미터 순서를 신경 쓸 필요가 없고, 어떤 필드에 어떤 값이 들어가는 지 명시적으로 확인할 수 있으며, 필드 추가 및 변경에도 용이합니다.

적용해서 수정하겠습니다!!